docs/pages/core_lib_doc/Data Layers/1.data_layers.md

```diff
@@ -45,7 +45,19 @@ class User(SoftDeleteMixin, Base):
     ...
 ```
 
-
+#### Code Explained:
+- <b>Imports:</b>
+    - Base: This is typically a SQLAlchemy declarative base class. It's a base class for all your model classes.
```

Comment:
declarative_base() [link to sqlalchemy ] It will use you can use it as a baseclass for all of your entities/models